### DISCUSSION TITLE
any timeframe on the new release?

### DISCUSSION THREAD

**Discussion by hemna** [sjlongland/aioax25#15] [General]
Just curious if/when you plan on a new release so I can get the fix for aprsd

**Comment by sjlongland** [comment]
I haven't got an immediate time-frame in mind at this stage… at the moment we've got two open issues:

* issue #11 relates to decoding of some position APRS messages.  MICe support is still absent at this stage, but most "conventional" APRS position reports should be readable and sendable now.
* issue #13 initially discussed an issue with sending APRS text messages with a user-supplied message ID, but in amongst that, there's also the issue of third-party APRS traffic, which isn't yet handled.

The last month I've been pretty busy, so haven't had a lot of time to work on packet radio stuff.  I guess the real question is what fixes are needed for `aprsd` right now, and what can wait until the next release?

**Reply by hemna** [reply]
I think the decoding of the 3rd party packets is a new feature and is non-trivial.  The sending of packets and let the application manage the msg number and resends/acks is a good fix.

**Reply by sjlongland** [reply]
https://pypi.org/project/aioax25/0.0.11/ is the new release.

Note that `pypi` didn't publish the digital signature, which triggered me to try and upload it a second time to ensure it wasn't something I missed.  It still didn't upload the digital signature.

So the `aioax25-0.0.11.tar.gz.asc` that goes with this release is here:
```
-----BEGIN PGP SIGNATURE-----

iQIzBAABCAAdFiEECPcoP9RNnUo1ljEuNdwZJ17JPjsFAmMvkqwACgkQNdwZJ17J
PjuDmRAAkhA3903xVSmTNrJXiXMcAI8QLtmyzofL1mAEFzA4CNfT5UbE5Jn+n2ro
CjxHX/NURjrnZ5CXR4f+jj/QmhUY7znJ3fw9RQ7Cl2/k/xfkdWwqvujsIholOvEu
KFrr+FWKcGMaMALJk1rsp9aAvZQ1wsJKXDhhzSXH6ivEPAuFC5aADwLDpedC0BR6
5vY1jWzWQh4Bw243F0kUtpCr/nG6YpINEfc7v+mW3MFeoO/QkfPoTr8wIKTcVeB5
EW7LjkolbiK1A/9y2nBLUMn2O+fx1cOtlvoD3HCQA5yh5xSdD86LG0sm2uD9pzkW
0s14CkAxWpNNF26xfSL5j2SYdzlPTt9cIcPVtC37ObmTpwuhtbSjUMDHeXNzfviA
nO02GYfFGl4lgkuBANjxi6+PVC+Pv2/7l1NJPVzgpJ9AoQndlQr3ql/2u0v/oQBH
ydAdyt8b7DkiIKDMcyoXdt6xf/ZmXqFz3kyiv0SZLyRmEyPisg8X0VdvwO9ubZC8
8ZS34V5qnRLFOsIEXi3y/qyZc49PUYZd34VD3A+C3Pa6tHT39JlgY/XeWXr1LmkG
8CZ9onE7wBZYMbJhSokTroySXN4KThmnxsO0yaBGxxnR5xo/9soofyzmBqLTS7zt
OED0YgbYmmrNbtP4yXutq6X2n/PueTEojiIfINGfHOG8A9Qs7bM=
=McuK
-----END PGP SIGNATURE-----
```
